src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,79 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+
+  const query = searchParams.get('query')?.toLowerCase() || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('century') || [];
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const sortPeople = [...people].sort((p1, p2) => {
+    switch (sort) {
+      case 'name':
+        return order
+          ? p1.name.localeCompare(p2.name)
+          : p2.name.localeCompare(p1.name);
+      case 'sex':
+        return order
+          ? p1.sex.localeCompare(p2.sex)
+          : p2.sex.localeCompare(p1.sex);
+      case 'born':
+        return order ? p1.born - p2.born : p2.born - p1.born;
+      case 'died':
+        return order ? p1.died - p2.died : p2.died - p1.died;
+      default:
+        return 0;
+    }
+  });
```

Comment:
You are sorting the people array every time the component re-renders. This could be a performance issue if the array is large. Consider using useMemo to memoize the sorted array and only recompute it when the 'people' or 'sort' props change.